kolibri/core/assets/src/views/PrivacyInfoModal.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+
+  <KModal
+    @cancel="$emit('cancel')"
+    size="large"
+    :cancelText="$tr('cancelButtonLabel')"
+    :title="$tr('privacyModalHeader')"
+  >
+    <section v-if="!hideUsersSection">
+      <h2>{{ $tr('kolibriUsersTitle') }}</h2>
+      <p>{{ $tr('kolibriUsersP1') }}</p>
+      <p>{{ $tr('kolibriUsersP2') }}</p>
+      <p>{{ $tr('kolibriUsersP3') }}</p>
+      <p>{{ $tr('kolibriUsersP4') }}</p>
+      <ul>
+        <li>{{ $tr('kolibriUsersL1') }}</li>
+        <li>{{ $tr('kolibriUsersL2') }}</li>
+        <li>{{ $tr('kolibriUsersL3') }}</li>
+        <li>{{ $tr('kolibriUsersL4') }}</li>
+      </ul>
+      <p>{{ $tr('kolibriUsersP5') }}</p>
+      <p>{{ $tr('kolibriUsersP6') }}</p>
+    </section>
+    <section v-if="!hideOwnersSection">
+      <h2>{{ $tr('kolibriOwnersTitle') }}</h2>
+      <p>{{ $tr('kolibriOwnersP1') }}</p>
+      <p>{{ $tr('kolibriOwnersP2') }}</p>
+      <p>{{ $tr('kolibriOwnersP3') }}</p>
+    </section>
+    <section v-if="!hideAboutSection">
+      <h2>{{ $tr('kolibriAboutTitle') }}</h2>
+      <p>{{ $tr('kolibriAboutP1') }}</p>
+      <p>
+        <KExternalLink
+          text="https://learningequality.org/"
+          href="https://learningequality.org/"
+          target="_blank"
+        />
+      </p>
+      <p>{{ $tr('kolibriAboutP2') }}</p>
+      <p>{{ $tr('kolibriAboutP3') }}</p>
+      <p>{{ $tr('kolibriAboutP4') }}</p>
+    </section>
+  </KModal>
+
+</template>
+
+
+<script>
+
+  import KExternalLink from 'kolibri.coreVue.components.KExternalLink';
+  import KModal from 'kolibri.coreVue.components.KModal';
+
+  export default {
+    name: 'PrivacyInfoModal',
+    components: {
+      KExternalLink,
+      KModal,
+    },
+    props: {
+      hideUsersSection: {
+        type: Boolean,
+        default: false,
+      },
+      hideOwnersSection: {
+        type: Boolean,
+        default: false,
+      },
+      hideAboutSection: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    $trs: {
+      privacyModalHeader: 'Usage and privacy',
+      cancelButtonLabel: 'Close',
+      kolibriUsersTitle: 'Users',
+      kolibriUsersP1:
+        'You should use Kolibri in compliance with all applicable laws. This may mean obtaining permission from your parent, guardian, or teacher.',
```

Comment:
Interested to know what these means in terms of translator experience (not that this is a blocker, but thinking how we could improve it).

Reply:
What's the concern?

The sentence seems pretty self-contained and renders nicely in crowdin. I'd considered doing the whole thing as a big Markdown-formatted block, but chose this route because I thought the translator experience would be better.